### PR TITLE
Specify that temp files are unneeded

### LINKED ISSUE #58: Not clear if deleting temporary files is safe
Onboarding asks users whether they want to automatically delete temporary files every 30 days. 
If this is a safe operation, I think it should be turned on by default (similar to how old kernels are now automatically pruned). If it isn't safe, then why are new users offered the option? If anyone should know which non-user files can be deleted, then it is the operating system.

If we want to offer this option, then I think it should be specific enough that the target audience understands what's going on: "We can automatically empty the /tmp directory if you are sure none of your apps uses it for important data". But that's super deep into techie land :)

1*Rncwyen7EHPQDWv4oWOkIA@2x

This is not actually a new issue, the same wording is used in the switchboard plug.

1*qWZHCeDnBkbDKvg7E0RZUw

### PR DESCRIPTION
**Pull request by dar5hak**
Closes #58.

### CODE REVIEW

**Comment by danrabbit** [comment]
Hrm, my first impression is that this simplification doesn't really clarify anything. If temporary files are "unneeded" then why create them in the first place?

I'm not sure if there's a concise way to explain caching in a checkbox label, but I'm not certain that this removes layers of confusion rather than adding more

@elementary/ux opinions?

**Comment by dar5hak** [comment]
@danrabbit yes, concise is the keyword there. "No longer needed" is more accurate but too wordy for something that most people won't uncheck anyway.

**Comment by lewisgoddard** [comment]
If this is only files older than 30 days, then I'd just say "Old Temporary Files".

File: src/Views/HouseKeepingView.vala [hunk 1]
```diff
@@ -31,7 +31,7 @@ public class Onboarding.HouseKeepingView : AbstractOnboardingView {
 
         var header_label = new Granite.HeaderLabel (_("Automatically Delete:"));
 
-        var temp_switch = new Gtk.CheckButton.with_label (_("Temporary files"));
+        var temp_switch = new Gtk.CheckButton.with_label (_("Unneeded temporary files"));
```
Comment (cassidyjames) [inline src/Views/HouseKeepingView.vala:34]:
```suggestion
        var temp_switch = new Gtk.CheckButton.with_label (_("Old temporary files"));
```

**Review by cassidyjames — APPROVED** [review]

**Comment by cassidyjames** [comment]
Thanks! This is small, but I do think it can improve the clarity here and increase the confidence of turning it on.
